patroni/dcs/__init__.py

```diff
@@ -1024,6 +1376,18 @@ def _get_citus_cluster(self) -> Cluster:
         return cluster
 
     def get_cluster(self, force: bool = False) -> Cluster:
+        """Retrieve an appropriate cached or fresh view of DCS.
+
+        .. note::
+            Stores copy of time, status and failsafe values for comparison in DCS update decisions.
+            This works around the unique operational requirement of Zookeeper to make extra queries for keys.
```

Comment:
```suggestion
            Caching is mostly required to avoid overhead placed on the REST API, due to the nature of how
            Zookeeper makes multiple requests are made in comparison to other DCS implementations.
```

Reply:
I wouldn't mention Zookeeper or any other DCS at all here.

TL;DR - there are two levels of caches:
1. `self._cluster` which is accessed via `cluster` property from REST API and cached only for `ttl` seconds.
2. sometimes DCS are have their own caches:
  - zookeeper does it for the entire Cluster
  - K8s and Etcd3 have caches for all keys associated with the cluster

The second type of caches exists to avoid reading something from DCS on every heart beat (when it is possible), and we shouldn't reveal here any implementation details.

Reply:
Updated it to a simpler form. Thanks for the explanation.

Reply:
:+1: